src/main/java/duke/logic/commands/ExportLockerCommand.java

```diff
@@ -0,0 +1,20 @@
+package duke.logic.commands;
+
+import duke.exceptions.DukeException;
+import duke.models.LockerList;
+import duke.storage.FileHandling;
+import duke.storage.OpenCsv;
+import duke.ui.Ui;
+
+public class ExportLockerCommand extends Command {
+
+    public ExportLockerCommand(){
```

Comment:
Try removing this constructor. I think we dont need to add this constructor as Java will automatically understand that you are calling the default constructor if you are not doing constructor overloading.

Reply:
resolved